packages/ui/stories/organisms/FloatingButton.stories.tsx

```diff
@@ -48,16 +48,33 @@ const actions = [
 
 storiesOf('ui/organisms/FloatingButton', module)
   .addDecorator(story => <View style={styles.decorator}>{story()}</View>)
```

Comment:
I thought you said decorator was not needed anymore ? 🤔

Reply:
If I don't leave this decorator this component would appear on the right, that's why